lib/DBus/TH.hs

```diff
@@ -0,0 +1,17 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE TemplateHaskell #-}
+module DBus.TH where
+
+import           DBus.Client
+import           DBus.Generation
+import qualified DBus.Introspection as I
+import           System.FilePath
+
+
+generateSignalsFromInterface defaultGenerationParams $
```

Comment:
Looks like generated code isn't very clean:
Top-level binding with no type signature: signalForPropertiesChanged :: DBus.Internal.Message.Signal
Top-level binding with no type signature: methodCallForInterfaces :: DBus.Internal.Message.MethodCall
Defined but not used: ‘replySuccess’
etc...

Can you make sure it compiles without warnings? It is very annoying when library adds a lot of warning noise.

Reply:
I got rid of all of these except for the replySuccess not being used. That one only comes up when there are no return values for a method. Getting rid of it would require a lot more code.